python/BUSINESS_SUPPORT_OR_ABOVE_ENABLED/business-support-enabled.py

```diff
@@ -0,0 +1,415 @@
+# Copyright 2017-2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may
+# not use this file except in compliance with the License. A copy of the License is located at
+#
+#        http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for
+# the specific language governing permissions and limitations under the License.
+'''
+  #####################################
+  ##           Gherkin               ##
+  #####################################  
+  Rule Name:
+  BUSINESS_SUPPORT_OR_ABOVE_ENABLED

```

Comment:
Tab it. Idem for the rest of the Gherkin. Example: https://github.com/awslabs/aws-config-rules/blob/master/python/ELASTICSEARCH_IN_VPC_ONLY/ELASTICSEARCH_IN_VPC_ONLY.py

Reply:
tabbed Gherkin as per given example